src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -766,69 +791,32 @@ include::insns/store_32bit_cap.adoc[]
 === Changes to Existing RISC-V Base ISA Instructions
 
 {cheri_base_ext_name} extend existing instructions that are used for handling addresses so that they manipulate a whole capability.
-The affected operands change type from *x* to *c* to represent this, and/or the PC is replaced by the <<pcc>>
-
-In the base ISA this affects <<AUIPC_CHERI>>, all jumps and branches as well as all loads and stores.
-The rules for modifying the behavior of such instructions are:
 
-* The result written to the destination *x* register is _always_ unchanged. I.e., the lower XLEN bits are always written with the value from RV32I/RV64I.
-* Whenever the PC is handled, it is _always_ the CLEN-bit <<pcc>>.
-* Any memory instruction that has `rs1` as a base address register reads the full capability register (`cs1`) instead.
-  The base address is unchanged, i.e., using the value from `rs1`.
-  The metadata and valid tag are used to <<sec_cap_checks,authorize the access>>.
+* The affected operands change type from *x* to *c* to represent this.
+* Whenever the address field of any capability is modified, it is _always_ updated using <<SCADDR>> semantics.
 * Any instruction that writes back an address (e.g., <<AUIPC_CHERI>> or <<CSRRW_CHERI>>) to the destination register, writes a full capability register instead.
-* Whenever the address field of any capability (including the <<pcc>>) is modified, it is _always_ updated using <<SCADDR>> semantics.
-  This includes adding an offset to the <<pcc>> from jumps and branches for both the target address and the link register.
-  In this case, e.g., `new_pcc = SCADDR(old_pcc, offset)`
-* All jumps and branches check that a minimum sized instruction is in bounds of the target <<pcc>>.
-  An exception is raised on the branch or jump on failure.
-* <<JALR_CHERI>> updates <<pcc>>.address as expected, but also copies the metadata and valid tag from `cs1` into the <<pcc>>.
-  If `cs1` is not suitable for execution then an exception is taken on the <<JALR_CHERI>>.
-  See xref:pcc[xrefstyle=full] for the definition of suitability.
 
 These rules affect the following <<rv32,base ISA>> instructions listed in <<tab_cap_base_summary>>, and also apply to instructions added by other extensions, e.g.:
```

Comment:
The table now only contains control flow instructions so it's not good to link to that.

Maybe just drop this link?
```suggestion
These rules affect the following <<rv32,base ISA>> instructions listed in the following subsections, and also apply to instructions added by other extensions, e.g.:
```